src/main/java/com/lothrazar/cyclic/block/harvester/IHarvesterOverride.java

```diff
@@ -0,0 +1,31 @@
+package com.lothrazar.cyclic.block.harvester;
```

Comment:
looks good to merge. but yeah you make a good point. should be in new folder

package com.lothrazar.cyclic.block.api;

Reply:
Do you want separate API packages in each of your mod packages instead of one separate API package?

Reply:
root of mod directory makes sense in api folder, like
src/main/java/com/lothrazar/cyclic/api/IHarvesterOverride.java 


Same idea as
https://github.com/Shadows-of-Fire/FastWorkbench/tree/1.16/src/main/java/shadows/fastbench/api

fastbench is mod and then whatever files we need can be in the api folder